apps/web/src/lib/components/ui/agent-editor/form-edit.svelte

```diff
@@ -26,14 +27,13 @@
 	};
 	$: isFormIncomplete =
 		!selectedAgent?.title || !selectedAgent?.role || !selectedAgent?.description;
-
 </script>
 
 <Dialog.Root {open} onOpenChange={handleChange}>
-	<Dialog.Content class="w-full sm:max-w-full lg:max-w-4xl">
-		<Dialog.Header>
+	<Dialog.Content class="w-full border-none sm:max-w-full lg:max-w-4xl">
+		<!-- <Dialog.Header>
```

Comment:
nit: avoid commented out code, it'll remain in git history so you can get it back later